tensorflow_lite_support/cc/task/vision/proto/landmarks.proto

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+syntax = "proto2";
+
+package tflite.task.vision;
+
+// A single detected landmark.
+message Landmark {
+  optional float key_y = 1;
+  optional float key_x = 2;

```

Comment:
Use an array to represent the position, because we may need to support 3D models in the future. So

message Landmark {
  repeated float position = 1;
  optional float score = 2;
}

Reply:
ok

Reply:
Just to be sure the array should look like landmark{y0,x0,score0,y1,x1,score1.......} correct?

Reply:
ignore score

Reply:
Just to sure array should look like position{y0,x0,y1,x1...} and score{s1,s2,s3...}?